kotlin/internal/jvm/compile.bzl

```diff
@@ -348,7 +341,7 @@ def _run_kt_builder_action(
 
     args.add_all("--kotlin_passthrough_flags", _kotlinc_options_provider_to_flags(toolchains.kt.kotlinc_options))
     args.add_all("--javacopts", _javac_options_provider_to_flags(toolchains.kt.javac_options))
-
+    args.add_all("--direct_dependencies", _java_infos_to_compile_jars(compile_deps.deps))
```

Comment:
Open an issue and todo this one -- it isn't used in the builder.

Reply:
Done